plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -242,14 +242,24 @@ func (c *PodSecurityPolicyPlugin) computeSecurityContext(a admission.Attributes,
 		podCopy := pod.DeepCopy()
 
 		if errs := assignSecurityContext(provider, podCopy, field.NewPath(fmt.Sprintf("provider %s: ", provider.GetPSPName()))); len(errs) > 0 {
+			// TODO: the error list may include errors from unauthorized PodSecurityPolicies.
```

Comment:
Can we do the authorization check to filter the list of policies iff the pod isn't allowed by any policy? That way permitted pods skip the additional check, but denied pods do the lookup?

Reply:
Yeah, I think that's the way to go. I'll add it today or tomorrow.